ext/easylogging-8.91/easylogging++.h

```diff
@@ -3133,6 +3168,13 @@ class Writer : private internal::NoCopy {
             if (logger_->typedConfigurations_->toStandardOutput(severity_)) {
                 std::cout << currLine_;
             }
+
+            for (Logger::Targets::iterator it = logger_->customTargets_.begin();
+                 it != logger_->customTargets_.end(); ++it) {
+                ILogTarget* target = *it;
+                target->log(currLine_);
+            }
+
             logger_->stream_->str("");
```

Comment:
If I correctly understand type of stream_ 
logger_->stream_->str(std::string()) should be better.